uber/config.py

```diff
@@ -502,6 +502,7 @@ def _is_intstr(s):
 c.EVENT_MONTH = c.EPOCH.strftime('%B')
 c.EVENT_START_DAY = int(c.EPOCH.strftime('%d')) % 100
 c.EVENT_END_DAY = int(c.ESCHATON.strftime('%d')) % 100
+c.EVENT_QR_ID = c.EVENT_QR_ID or c.EVENT_NAME_AND_YEAR.replace(' ', '_').lower()
```

Comment:
By the way, the result of this for our current config is just `magfest`, so we either want to set the year or configure a custom event_qr_id.